diff --git a/.bazelversion b/.bazelversion
index 3eefcb9..0b2eb36 100644
--- a/.bazelversion
+++ b/.bazelversion
@@ -1 +1 @@
-1.0.0
+3.7.2
diff --git a/bazel/zetasql_deps_step_2.bzl b/bazel/zetasql_deps_step_2.bzl
index 1a74849..168a24a 100644
--- a/bazel/zetasql_deps_step_2.bzl
+++ b/bazel/zetasql_deps_step_2.bzl
@@ -107,9 +107,9 @@ cc_proto_library(
         http_archive(
             name = "com_google_absl",
             # Commit from 2021-02-23
-            url = "https://github.com/abseil/abseil-cpp/archive/a50ae369a30f99f79d7559002aba3413dac1bd48.tar.gz",
-            sha256 = "be2a9d7ea7ee15f9317b57beff37e8ffb67418fb0df64592366b04c8618c2584",
-            strip_prefix = "abseil-cpp-a50ae369a30f99f79d7559002aba3413dac1bd48",
+            url = "https://github.com/abseil/abseil-cpp/archive/2e94e5b6e152df9fa9c2fe8c1b96e1393973d32c.zip",
+            sha256 = "485f7488f2102edd702d1cac1c30f30efa6f3bc793999aeb92f161b2dbe707dd",
+            strip_prefix = "abseil-cpp-2e94e5b6e152df9fa9c2fe8c1b96e1393973d32c",
         )
 
     # Abseil (Python)
